smarsy/parse.py

```diff
@@ -135,29 +135,42 @@ def bs_safeget(soup, selector):
     return False
 
 
-def parent_page_content_to_object(html):
+def create_parents_dict(parent_html) -> dict:
+    """
+    Function receive HTML and return parent dictionary with given keys
+    """
+    parent_dict = {}
+    parent_list = []
+    parents_keys = ['parent_img', 'parent_name', 'parent_surname',
+                    'parent_middlename', 'parent_type',
+                    'parent_birth_date']
+    parent_img = bs_safeget(parent_html, '[valign=top]').img['src']
+    if parent_img:
+        parent_list.append(parent_img)
+    parents_name = bs_safeget(parent_html, '.username').text.split(' ')
+    if parents_name:
+        parent_list += [value for value in (
```

Comment:
previous implementation was easier to understand. Here is not clear what is going on. I mean not technically, but logically. It should be moved to it's own function

Reply:
Can I do like this?
parent_list.append(parents_name[0])
parent_list.append(parents_name[1])
....